scenarios/arcaflow/memory-hog/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
we might want to make these configs the entry point i.e have these replace litmus based hog scenario configs under scenarios/. Also update https://github.com/redhat-chaos/krkn#kubernetesopenshift-chaos-scenarios-supported with specific hog scenarios instead of litmus/arcaflow for easy entry point for folks to understand scenarios supported. Thoughts?

Reply:
Tried to make the requested changes in the Readme files please check if are ok.
Not sure on how to replace litmus scenarios in scenarios folder, could you give me a more specific example?
🙏

Reply:
Changes made should cover it. Thanks.